src/Symfony/Component/Messenger/composer.json

```diff
@@ -17,12 +17,13 @@
     ],
     "require": {
         "php": "^7.2.5",
-        "psr/log": "~1.0"
+        "psr/log": "~1.0",
+        "symfony/messenger-amqp": "^5.1",
+        "symfony/messenger-doctrine": "^5.1",
+        "symfony/messenger-redis": "^5.1"
```

Comment:
Should'nt we trigger a deprecation somehow telling `the "XXX" transport has been moved in the package "symfony/messenger-xxx" and will not be included by default in 6.0, run composer require "symfony/messenger-xxx"`?